src/libs/ReportUtils.ts

```diff
@@ -4155,9 +4155,27 @@ function doesTransactionThreadHaveViolations(report: OnyxEntry<Report>, transact
     if (report?.stateNum !== CONST.REPORT.STATE_NUM.OPEN && report?.stateNum !== CONST.REPORT.STATE_NUM.SUBMITTED) {
         return false;
     }
+
```

Comment:
Unless this was added to comply with the style checker, let's remove it. Blank lines should be preceded by comments
```suggestion
```